### ISSUE TITLE
Shouldn't the timezone be considered?

### ISSUE THREAD

**Issue by deostroll** [sebbo2002/ical-generator#49]
First off, nice job...

But I think you may have an error at this line...

https://github.com/sebbo2002/ical-generator/blob/develop/lib/_tools.js#L44

The `formatDateTZ` function should actually return the date/time as per target timezone, (i.e., the calendar timezone). However, the string returned gives the UTC time which I think is probably a mistake.

**Comment by takkaria** [comment]
Yeah, at the moment timezone support is fudged a bit (I wrote this bit).  Using UTC is obviously not correct, but neither is using the local time set on the Date object, because the timezone being output in the ical file will not necessarily be the machine's local date.  It seemed to be more straightforward and predictable to use UTC dates in this case.

I think the solution would be to pull in moment-timezone as a dependency for proper timezone handling.  I started sketching this out at on #38 but maybe I should finish it off.

**Comment by sebbo2002** [comment]
As long as using UTC datetimes works in nearly every client I would stick to this solution, because it's not possible to detect the end users location and timezone right now.

Did you have any issues with UTC dates?

**Comment by ocelotsloth** [comment]
+1 to using `moment-timezone`. It's what I'm using in git.gmu.edu/srct/schedules and it's a bit hard to follow what happens to the timezones and the times themselves between going into and out `ical-generator`.

**Comment by sebbo2002** [comment]
ical-generator 3.0 now uses moment-timezone, so this issue should be fixed. I also added the above example in the [unit tests](https://github.com/sebbo2002/ical-generator/blob/develop/test/test_issues.js#L8). You can test the current develop with `npm i ical-generator@next`.
